### ISSUE TITLE
cpuTemperature does not parse negative temperatures correctly

### ISSUE THREAD

**Issue by jewalt** [sebhildebrandt/systeminformation#140]
### Issue Type

* [X ] Bug
* [ ] Feature Request
* [ ] Question

### Expected behavior 
Negative temperatures from sensors output should be parsed and returned as negative numbers

### Actual behavior
temperature is always +90 when the CPU temperature is negative

### Steps to reproduce the problem
Get the CPU really cold.

### Specifications 

- package version:
3.32.1
- operating system:
Linux
- hardware:
Intel Joule

### Problem and possible solution
With the sensors output:
```
sensors
coretemp-isa-0000
Adapter: ISA adapter
Physical id 0:   -5.0°C  (high = +90.0°C, crit = +90.0°C)
Core 0:          -5.0°C  (high = +90.0°C, crit = +90.0°C)
Core 1:          -5.0°C  (high = +90.0°C, crit = +90.0°C)
Core 2:          -6.0°C  (high = +90.0°C, crit = +90.0°C)
```
From an Intel Joule running Ubuntu 16.04, I would have expected to receive a negative core temp.  Instead I received:
`data: { main: 90, cores: [ 90, 90, 90 ], max: 90 }`

Obviously due to the high and crit temperatures in the output.

Could the regex in line 465 be changed to:
`let regex = /[\+-]([^°]*)/g;`

This would match on positive and negative temperatures.  I have done some basic testing on my system and this does solve the problem.

**Comment by sebhildebrandt** [comment]
Fixed ... my fault ... just published version 3.42.4

**Comment by jewalt** [comment]
Thanks Sebastian!
